theano/tensor/basic.py

```diff
@@ -4573,7 +4573,7 @@ def reshape(x, newshape, ndim=None, name=None):
                 "to know what the number of dimensions of the reshaped "
                 "variable will be. You can provide the 'ndim' keyword "
                 "argument to 'reshape' to avoid this problem." % newshape)
-    op = Reshape(ndim, name)
+    op = Reshape(ndim)
```

Comment:
Run all Theano tests on a computer with a GPU. There is some code in GPU that support it is there. Look at least in theano/sandox/cuda/basic_ops.py theano/sandbox/cuda/opt.py and theano/gpuarray/basic_ops.py